### PR TITLE
New rule definition to support instant and aggregation time window

### PR DESCRIPTION
**Pull request by zhang-hua**
# Type of change? 

- [ ] Bug fix
- [x] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 

New rule definition to support instant and aggregation time window

**Checklist:**

- [x] All tests passed
- [x] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: Services/Data/default.json [hunk 1]
```diff
@@ -69,6 +69,8 @@
       "Description": "Pressure > 250",
```
Comment (ppathan) [inline Services/Data/default.json]: Use the [default.json](https://github.com/Azure/device-telemetry-dotnet/blob/master/Services/Data/Rules/default.json) from device-telemetry-dotnet.

File: Services/Data/default.json [hunk 2]
```diff
@@ -93,27 +95,29 @@
...
-      "GroupId": "default_Engines",
-      "Severity": "info",
+      "GroupId": "Engines",
+      "Severity": "Info",
+      "Calculation": "Instant",
+      "TimePeriod": "00:00:00",
       "Conditions": [
         {
-          "Field": "fuellevel",
+          "Field": "fuel",
```
Comment (ppathan) [inline Services/Data/default.json]: This should be actually `fuellevel`

File: Services/Data/default.json [hunk 3]
```diff
@@ -63,12 +63,13 @@
   ],
   "Rules": [
     {
-      "Id": "default_Chiller_Pressure_High",
       "Name": "Chiller pressure too high",
       "Enabled": true,
       "Description": "Pressure > 250",
-      "GroupId": "default_Chillers",
```
Comment (ppathan) [inline Services/Data/default.json]: Can you revert the GroupId values to what we had before prefix of `default_`?

File: Services/Data/default.json [hunk 4]
```diff
@@ -63,12 +63,13 @@
   ],
   "Rules": [
     {
-      "Id": "default_Chiller_Pressure_High",
```
Comment (ppathan) [inline Services/Data/default.json]: Revert, we need Id field

File: Services/Data/default.json (hunk 2 above)
Comment (zhang-hua) [reply Services/Data/default.json]: should we change this for telemetry service too?

File: Services/Data/default.json (hunk 3 above)
Comment (zhang-hua) [reply Services/Data/default.json]: should we change this for telemetry service too?

File: Services/Data/default.json (hunk 4 above)
Comment (zhang-hua) [reply Services/Data/default.json]: should we change this for telemetry service too?

Comment (ppathan) [reply Services/Data/default.json]: Actually the one in telemetry service should be deleted to avoid confusion as it is not being used.

**Pull request merged**